doc/source/ray-overview/installation.rst

```diff
@@ -49,16 +49,43 @@ You can install the nightly Ray wheels via the following links. These daily rele
-=============================================== ================================================  ====================  =======================
-`Linux Python 3.10 (x86_64)`_                    `Linux Python 3.10 (aarch64)`_                   `MacOS Python 3.10`_  `Windows Python 3.10`_
-`Linux Python 3.9 (x86_64)`_                     `Linux Python 3.9 (aarch64)`_                    `MacOS Python 3.9`_   `Windows Python 3.9`_
-`Linux Python 3.8 (x86_64)`_                     `Linux Python 3.8 (aarch64)`_                    `MacOS Python 3.8`_   `Windows Python 3.8`_
-`Linux Python 3.7 (x86_64)`_                     `Linux Python 3.7 (aarch64)`_                    `MacOS Python 3.7`_   `Windows Python 3.7`_
-`Linux Python 3.6 (x86_64)`_                     `Linux Python 3.6 (aarch64)`_                    `MacOS Python 3.6`_
-`Linux Python 3.11 (x86_64) (EXPERIMENTAL)`_     `Linux Python 3.11 (aarch64) (EXPERIMENTAL)`_
-=============================================== ================================================  ====================  =======================
+
+.. tabbed:: Linux
+
+    =============================================== ================================================  
+           Linux (x86_64)                                   Linux (arm64/aarch64)                     
+    =============================================== ================================================  
+    `Linux Python 3.10 (x86_64)`_                    `Linux Python 3.10 (aarch64)`_                   
+    `Linux Python 3.9 (x86_64)`_                     `Linux Python 3.9 (aarch64)`_                    
+    `Linux Python 3.8 (x86_64)`_                     `Linux Python 3.8 (aarch64)`_                    
+    `Linux Python 3.7 (x86_64)`_                     `Linux Python 3.7 (aarch64)`_                    
+    `Linux Python 3.6 (x86_64)`_                     `Linux Python 3.6 (aarch64)`_                    
+    `Linux Python 3.11 (x86_64) (EXPERIMENTAL)`_     `Linux Python 3.11 (aarch64) (EXPERIMENTAL)`_
+    =============================================== ================================================  
+
+.. tabbed:: MacOS
+
+    ================================  ================================
+     MacOS (x86_64)                    MacOS (arm64)
+    ================================  ================================
+    `MacOS Python 3.10 (x86_64)`_      `MacOS Python 3.10 (arm64)`_
+    `MacOS Python 3.9 (x86_64)`_       `MacOS Python 3.9 (arm64)`_
+    `MacOS Python 3.8 (x86_64)`_       `MacOS Python 3.8 (arm64)`_
+    `MacOS Python 3.7 (x86_64)`_
+    `MacOS Python 3.6 (x86_64)`_
+    ================================  ================================
+
+.. tabbed:: Windows (beta)
+
+    .. list-table:: Title
+        :header-rows: 1
+
+       * - Windows (beta)
+       * - `Windows Python 3.10`_
+       * - `Windows Python 3.9`_
+       * - `Windows Python 3.8`_
+       * - `Windows Python 3.7`_
```

Comment:
Sphinx does not support single-column tables in simple format, so we have to go for this format instead.